Upstream: Gentoo wants a Gentoo branding, of course.
Source: philantrop@exherbo.org
Summary: Remove the more prominent Gentoo brandings and replace them with
Exherbo ones.

diff -Naur baselayout-1.12.12.orig/etc/issue.devfix baselayout-1.12.12/etc/issue.devfix
--- baselayout-1.12.12.orig/etc/issue.devfix	2007-10-24 00:59:26.000000000 +0200
+++ baselayout-1.12.12/etc/issue.devfix	2008-05-23 12:56:46.000000000 +0200
@@ -1,7 +1,7 @@
 -----------------------------------------------------
 Your system seems to be missing critical device files 
-in /dev !  Although you may be running udev or devfs, 
-the root partition is missing these required files !
+in /dev!  Although you may be running udev or devfs, 
+the root partition is missing these required files!
 
 To rectify this situation, please do the following:
 mkdir /mnt/fixit
@@ -16,6 +16,5 @@
 your system, you will have to restore your old issue 
 file in order to get rid of this warning.
 
-Thanks for using Gentoo ! :)
 http://bugs.gentoo.org/show_bug.cgi?id=40987
 -----------------------------------------------------
diff -Naur baselayout-1.12.12.orig/sbin/rc baselayout-1.12.12/sbin/rc
--- baselayout-1.12.12.orig/sbin/rc	2007-10-24 00:59:27.000000000 +0200
+++ baselayout-1.12.12/sbin/rc	2008-05-23 12:56:01.000000000 +0200
@@ -63,7 +63,7 @@
 		if ! mkdir -p "$1" &>/dev/null ; then
 			splash "critical" &
 			echo
-			eerror "For Gentoo to function properly, \"$1\" needs to exist."
+			eerror "For Exherbo to function properly, \"$1\" needs to exist."
 			if [[ ${RC_FORCE_AUTO} == "yes" ]] ; then
 				eerror "Attempting to create \"$1\" for you ..."
 				mount -o remount,rw /
@@ -207,13 +207,13 @@
 	${RC_DMESG_LEVEL+/bin/dmesg -n ${RC_DMESG_LEVEL}}
 
 	echo
-	echo -e "${GOOD}Gentoo Linux${GENTOO_VERS}; ${BRACKET}http://www.gentoo.org/${NORMAL}"
+	echo -e "${GOOD}Exherbo ${BRACKET}http://www.exherbo.org/${NORMAL}"
 	echo -e " Copyright 1999-2007 Gentoo Foundation; Distributed under the GPLv2"
 	echo
 	if [[ ${RC_INTERACTIVE} == "yes" ]] ; then
 		echo -e "Press ${GOOD}I${NORMAL} to enter interactive boot mode"
 		echo
-    fi
+    	fi
 	check_statedir /proc
 
 	ebegin "Mounting proc at /proc"
@@ -327,15 +327,10 @@
 		# OK, if we got here, things are probably not right :)
 		if [[ ${devfs} == "no" && ${udev} == "no" ]] ; then
 			echo
-			einfo "The Gentoo Linux system initialization scripts have detected that"
-			einfo "your system does not support UDEV.  Since Gentoo Linux has been"
-			einfo "designed with dynamic /dev in mind, it is highly suggested that you"
-			einfo "emerge sys-fs/udev and configure your system to use it."
-			einfo "Please read the Gentoo Handbook for more information!"
-			echo
-			einfo "    http://www.gentoo.org/doc/en/handbook/"
-			echo
-			einfo "Thanks for using Gentoo! :)"
+			einfo "The system initialization scripts have detected that your system"
+			einfo "does not support UDEV.  Since this distribution has been designed"
+			einfo "with dynamic /dev in mind, it is highly recommended that you"
+			einfo "install sys-fs/udev and configure your system to use it."
 			echo
 			read -t 15 -p "(hit Enter to continue or wait 15 seconds ...)"
 		fi
